src/styles/jsTreeTheme.less

```diff
@@ -95,8 +95,30 @@ li.jstree-closed > ul { display:none; }
 .jstree ins {
     position: absolute;
 }
+@extensionName: gif, html, svg;
 ins.jstree-icon {
-    left: -@jstree-icon-backindent !important;
+    left: @jstree-icon-backindent !important;
+    background: url("images/icons/default.gif") center center no-repeat !important;
```

Comment:
I'm pretty sure we're going to want SVG vs. GIF for these.  We should get icons from @flukeout